homework-submission/week3_app/src/components/FetchForeign.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+const DisplayUser = () => {
+  const [loding, setLoding] = useState(true);
```

Comment:
It's 'loading' instead of 'loding', if you aren't sure about something in English try to always check it, so that it doesn't confuse other developers who have to work with your code.

Reply:
I missed that spelling mistake.
thanks.